# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpfe/appshell/nsIXULBrowserWindow.idl
# Commit: 6955906262c0
# Full Hash: 6955906262c0920b0f9f28dcf3177e9ce5e0dffc
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-07 03:53:09
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 8: Remove forceInitialBrowserNonRemote, r=Gijs
#   
#   This method is no longer necessary, as frontend will always make the correct
#   decisions for the remoteness of the intial content browser by reading
#   openWindowInfo.
# ==============================================================================

diff -r a7700472807a -r 6955906262c0 xpfe/appshell/nsIXULBrowserWindow.idl
--- a/xpfe/appshell/nsIXULBrowserWindow.idl	Mon Apr 06 14:30:19 2020 +0000
+++ b/xpfe/appshell/nsIXULBrowserWindow.idl	Mon Apr 06 14:30:22 2020 +0000
@@ -43,13 +43,6 @@
                                 in boolean isAppTab);
 
   /**
-   * Find the initial browser of the window and set its remote attributes.
-   * This can be used to ensure that there is a browser in a new window of the
-   * correct type when it first spawns.
-   */
-  void forceInitialBrowserNonRemote(in mozIDOMWindowProxy openerWindow);
-
-  /**
    * Determines whether a load should continue.
    *
    * @param aDocShell
